### DISCUSSION TITLE
It runs great on Gmod but I kinda die a lot and some things are broken

### DISCUSSION THREAD

**Discussion by BAKERAP22** [Joshua-Ashton/VPhysics-Jolt#84] [General]
I can have hundreds of props easy, but things like vehicles and thrusters are all broken. Thrusters seem to be more powerful with the same amount of force than they used to be, and driving vehicles kills me either very fast or very suddenly and I get thrown across the map. Also touching things sometimes kills me and sends me flying. However, the engine is still just 0.10, and I understand that the engine technically isn't optimized for Gmod, so that is to be expected. I'm really liking this so far, this has a LOT of potential. You could make Source games run so much smoother for so many people.

**Comment by misyltoad** [comment]
The vehicle death thing is already fixed, development builds can be found here

https://git.froggi.es/joshua/vphysics_jolt_gmod_builds

**Reply by BAKERAP22** [reply]
Oh, excellent. Hush my mouth, then.

**Reply by BAKERAP22** [reply]
I hate to be a bother but I still seem to die a lot from driving. Any small ramp or bump and I either take a bunch of damage or immediately die and get sent at 4,000 miles per hour out of the vehicle. Also the airboat doesn't float anymore. I managed to drive it into the water without dying and it sank like a rock.

**Reply by misyltoad** [reply]
Try the 0.11 release.
